README.md

```diff
@@ -1,2 +1,3 @@
-# kyma-bundles
-Kyma helm broker bundles
+# Bundles
+
+Bundles repository contains sources of bundles which are used by the Helm-Broker.
```

Comment:
`` `Bundles` repository contains sources of bundles which the Helm Broker uses.``

Also: this README needs to be elaborated on